pkg/restapi/issuer/operation/operations.go

```diff
@@ -383,6 +384,18 @@ func (o *Operation) createIssuerProfileHandler(rw http.ResponseWriter, req *http
 		return
 	}
 
+	if profileData.SupportsWACI {
+		for _, pCredScope := range profileData.CredentialScopes {
+			if _, ok := o.cmOutputDescriptor[pCredScope]; !ok {
+				commhttp.WriteErrorResponseWithLog(rw, http.StatusInternalServerError,
+					fmt.Sprint("failed to find the allowed cred scope in credential manifest output descriptor "+
```

Comment:
Its better to pass the missing scope in the response message as to be more explicit about failed scope to the caller.

Reply:
One other thing I'd like to suggest...

Is this scenario something that can happen due to an incorrect configuration by the user? If so - you can add something here to make it more explicit how it happens and give them a hint as to how to fix it. (Something like `are the scopes configured?`)

However, if that configuration is already validated and this error is only here because of the `ok` check (and so this scenario is essentially impossible) then you can ignore what I wrote above.

Reply:
By impossible, I mean that there's no way a user can end up in this situation without modifying the code somehow.

Reply:
configuration is already validated and this error is only here because of the ok check.